pkg/controller/elasticsearch/validation/validations.go

```diff
@@ -34,6 +34,7 @@ const (
 	parseStoredVersionErrMsg = "Cannot parse current Elasticsearch version. String format must be {major}.{minor}.{patch}[-{label}]"
 	parseVersionErrMsg       = "Cannot parse Elasticsearch version. String format must be {major}.{minor}.{patch}[-{label}]"
 	pvcImmutableErrMsg       = "volume claim templates can only have their storage requests increased, if the storage class allows volume expansion. Any other change is forbidden"
+	pvcNotMounted            = "volume claim declared but volume not mounted in any container"
```

Comment:
Should we make it more explicit to cover the usual user mistake?
`volume claim declared but volume not mounted in any container. Please note the Elasticsearch data volume should be named "elasticsearch-data"`